conans/assets/templates/search_table_html.py

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
This should be an actual `.html` file installed together with Conan into the library folder.

Reply:
The main problem with that is how error prone is to create the installers with pyinstaller to embed and use other resources than code. It is tricky, and installers aren't that tested after the creation, so it is easy to have broken installers.